sdks/java/core/src/test/java/org/apache/beam/sdk/io/CompressedSourceTest.java

```diff
@@ -83,7 +82,7 @@
 @RunWith(JUnit4.class)
 public class CompressedSourceTest {
 
-  private final double DELTA = 1e-6;
+  private final double delta = 1e-6;
```

Comment:
nit: you should have declared this static and kept the capital letters instead of making it a member variable of CompressedSourceTest